public/script.js

```diff
@@ -1,4 +1,32 @@
-let CONSTANTS;
+// Packages
+import $ from 'jquery';
+window.jQuery = $;
+window.$ = $;
+
+import * as util from './mapUtil.js';
+
+import moment from 'moment';
```

Comment:
Does javascript have a convention for import order?

e.g., in python you place standard library imports at the top, then 3rd party imports, and finally project-specific imports (from other modules in this project) last.

```
import os

import numpy

import x from myproject
```